packages/lesswrong/server/scripts/convertImagesToCloudinary.ts

```diff
@@ -159,7 +159,7 @@ function getImageUrlsFromImgTag(tag: any): string[] {
   }
   const srcset: string = tag.attr("srcset");
   if (srcset) {
-    const imageVariants = srcset.split(",").map(tok=>tok.trim());
+    const imageVariants = srcset.split(", ").map(tok=>tok.trim());
```

Comment:
```suggestion
    const imageVariants = srcset.split(",\s").map(tok=>tok.trim());
```

Would it be valid html have only a newline after the comma?

Reply:
And below